package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "pinterest-for-woocommerce",
+  "title": "Pinterest for WooCommerce",
   "description": "Pinterest for WooCommerce",
   "version": "1.0.0-beta.4",
```

Comment:
Got a warning:

> WR [11:44:29] [WARNING] No version declared in package.json for pinterest-for-woocommerce. Skipping...

The `.4` parts in the `1.0.0-beta.4` can not match [the parsing pattern in woorelease](https://github.com/woocommerce/woorelease/blob/2.0.4/includes/tools/class-version-bump.php#L253). Probably needs to give a temporary placeholder or maybe change to `1.0.0-rc`.

The same goes for the `1.0.0-beta.1` at line #3 in the [package-lock.json file](https://raw.githubusercontent.com/woocommerce/pinterest-for-woocommerce/5aef86fa169dd1259ce67dcd26b53bfdf8a5aa08/package-lock.json), otherwise it will accidentally match the version of `@babel/code-frame` below.

![image](https://user-images.githubusercontent.com/17420811/138274596-050f6d05-5679-44a1-87a2-da06b5b21580.png)

Reply:
Thanks @eason9487 - we'll be updating to 1.0.0 so I think we'll be gtg